crates/mpt/src/node.rs

```diff
@@ -361,6 +495,20 @@ fn rlp_list_element_length(buf: &mut &[u8]) -> alloy_rlp::Result<usize> {
     Ok(list_element_length)
 }
 
+/// Unpack node path to nibbles.
+///
+/// ## Takes
+/// - `first` - first nibble of the path if it is odd. Must be <= 0x0F, or will create invalid
+///   nibbles.
+/// - `rest` - rest of the nibbles packed
+///
+/// ## Returns
+/// - `Nibbles` - unpacked nibbles
+fn unpack_path_to_nibbles(first: Option<u8>, rest: &[u8]) -> Nibbles {
+    let rest = Nibbles::unpack(rest);
+    Nibbles::from_vec_unchecked(first.into_iter().chain(rest.iter().copied()).collect::<Vec<u8>>())
+}
```

Comment:
Wonder if this should just be a macro. Preference?

Reply:
Would be neat to do: `nibbles!("646f")`

Reply:
Ah yeah, maybe not for this function since it's specific to unpacking path nibbles, but that'd be nice. Maybe we should make a PR into `nybbles` for that :D